src/site/content/en/reliable/broadcast-updates-guide/index.md

```diff
@@ -0,0 +1,282 @@
+  <img src="squoosh-screenshot.png"
+       width="550"
+       alt="A screenshot of Squoosh webapp 'Ready to work offline' functionality.">
+  <figcaption class="w-figcaption">In the Sqoosh PWA the service worker broadcasts an update to the page when cache is ready, and the page displays “Ready to work offline” toast.
+</figcaption>
+</figure>
+
+## Using Workbox
+
+### Listen to service worker lifecycle events
+
+Workbox Window provides a straightforward interface to listen to [important service worker lifecycle
+events](https://developers.google.com/web/tools/workbox/modules/workbox-window#important_service_worker_lifecycle_moments).
+Under the hood, the library uses client-side APIs like
+[onupdatefound](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/onupdatefound)
+and [onstatechange](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorker/onstatechange)
+and provides higher level event listeners in the Workbox Window object, making it easier for the
+user to consume these events.
+
+The following page code lets you detect every time a new version of the service worker is installed,
+so you can communicate it to the user:
+
+```javascript
+const wb = new Workbox('/sw.js');
+
+wb.addEventListener('installed', (event) => {
+  if (event.isUpdate) {
+    // Show “Update App” banner
+  }
+});
+
+wb.register();
+```
+
+### Inform the page of changes in cache data
+
+The Workbox package
+[workbox-broadcast-update](https://developers.google.com/web/tools/workbox/modules/workbox-broadcast-update)
+provides a standard way of notifying window clients that a cached response has been updated. This is
+most commonly used along with the [StaleWhileRevalidate
+strategy](https://developers.google.com/web/tools/workbox/modules/workbox-strategies#stale-while-revalidate).
+
+To broadcast updates add a `broadcastUpdate.BroadcastUpdatePlugin` to your strategy options in the
+service worker side:
+
+```javascript
+import {registerRoute} from 'workbox-routing';
+import {StaleWhileRevalidate} from 'workbox-strategies';
+import {BroadcastUpdatePlugin} from 'workbox-broadcast-update';
+
+registerRoute(
+  ({url}) => url.pathname.startsWith('/api/'),
+  new StaleWhileRevalidate({
+    plugins: [
+      new BroadcastUpdatePlugin(),
+    ],
+  })
+);
+```
+
+In your web app, you can listen for these events like so:
+
+```javascript
+navigator.serviceWorker.addEventListener('message', async (event) => {
+  // Optional: ensure the message came from workbox-broadcast-update
+  if (event.data.meta === 'workbox-broadcast-update') {
+    const {cacheName, updatedUrl} = event.data.payload;
+
+    // Do something with cacheName and updatedUrl.
+    // For example, get the cached content and update
+    // the content on the page.
+    const cache = await caches.open(cacheName);
+    const updatedResponse = await cache.match(updatedUrl);
+    const updatedText = await updatedResponse.text();
+  }
+});
+```
+
+## Using Browser APIs
```

Comment:
```suggestion
## Using browser APIs
```
Plz double-check that all your section headings are sentence case, not title case